platform/solr/platform-solr-server-standalone/src/main/resources/solr/conf/schema.xml

```diff
@@ -391,6 +392,7 @@
             <tokenizer class="solr.WhitespaceTokenizerFactory"/>
             <filter class="solr.PatternReplaceFilterFactory"
                     pattern="^\p{Punct}*(.*?)\p{Punct}*$" replacement="$1"/>
+            <filter class="solr.SynonymFilterFactory" synonyms="synonyms.txt" ignoreCase="true" expand="true"/>
```

Comment:
❓ should ingoreCase be false here, since this is a case-sensitive field?

Reply:
probably not - good catch